packages/api-docs-builder/utils/replaceUrl.test.js

```diff
@@ -222,13 +222,13 @@ describe('replaceUrl', () => {
     expect(replaceAPILinks(`/api/no-ssr/`)).to.equal(`/base/api/no-ssr/`);
     expect(replaceAPILinks(`/api/portal/`)).to.equal(`/base/api/portal/`);
     expect(replaceAPILinks(`/api/textarea-autosize/`)).to.equal(`/base/api/textarea-autosize/`);
-    expect(replaceAPILinks(`/api/button-unstyled/`)).to.equal(`/base/api/button-unstyled/`);
+    expect(replaceAPILinks(`/api/button-unstyled/`)).to.equal(`/base/api/button/`);
```

Comment:
I don't know where this is used but these urls are no longer valid, check the _redirection file.

Reply:
addressed